includes/class-boldgrid-editor-assets.php

```diff
@@ -194,9 +219,13 @@ public function front_end() {
 			)
 		);
 
-		wp_enqueue_style( 'animatecss',
-			plugins_url( '/assets/css/animate.min.css', BOLDGRID_EDITOR_ENTRY ),
-			array(), BOLDGRID_EDITOR_VERSION );
+		$maybe_enqueue_animate = $this->maybe_enqueue_animate();
+
+		if ( $maybe_enqueue_animate ) {
```

Comment:
$maybe_enqueue_animate is not really needed. What if we consolidated the code?

`if ( $this->maybe_enqueue_animate() ) {`